sys/include/net/gnrc/rpl/structs.h

```diff
@@ -28,6 +28,7 @@ extern "C" {
 #endif
 
 #include "net/ipv6/addr.h"
+#include "vtimer.h"
```

Comment:
an error in RPL itself. "vtimer.h" was only included by trickle, instead of including it itself. So when I removed vtimer from trickle, RPL complained